adoc/bp_chap_expanded_support.adoc

```diff
@@ -383,3 +383,130 @@ See the matrix of {productname} clients on the main page of the {productname} wi
 {prompt.root}systemctl restart salt-minion
 ----
 . Proceed to menu:Main Menu[Salt > Keys] from the {webui} and accept the minion's key.
+
+
+
+[[bp.expanded-support.ubuntu]]
+== Managing {ubuntu} Clients
+
+
+Support for {ubuntu} Clients was added in {susemgr} 3.2.
+Currently, Salt minions running {ubuntu} 16.04 LTS and 18.04 are supported.
+
+Bootstrapping is supported for starting {ubuntu} clients and performing initial state runs such as setting repositories and performing profile  updates.
+However, {ubuntu} lacks a root user by default, so in order to use bootstrapping, you will require an existing user with [command]``sudo`` privileges for Python.
+
+Other supported features:
+
+* Synchronizing [systemitem]``.deb`` channels
+* Assigning [systemitem]``.deb`` channels to minions
+* GPG signing [systemitem]``.deb`` repositories
+* Minion cleanup on delete
+* Information displayed in System details pages
+* Package install, update, and remove
+* Package install using [command]``Package States``
+* Configuration & state channels
+
+Some actions are not yet supported, however these may become available in a later version:
+
+* Patch and errata support
+* Bare metal installations, PXE booting, and virtual host provisioning
+* Live patching
+
+Note that {ubuntu} clients must be Salt minions.
+Traditional clients running {ubuntu} are not supported.
+
+This section describes how to manually prepare {ubuntu} clients for registration to your {susemgr} Server and synchronize the repositories.
+
+
+.Procedure: Preparing an {Ubuntu} 16.04 Client for Registration
+
+. On the client, open the [filename]``/etc/apt/sources.list.d/suma_client_tools.list`` file in your preferred text editor, and add this line to the bottom of the file:
```

Comment:
"preferred" -- super superfluous ;) I'd even say that this would be sufficient:

> On the client, add this line to the bottom of the the [filename]``/etc/apt/sources.list.d/suma_client_tools.list`` file: